### ISSUE TITLE
Question about Fig. 4 in the latest arxiv version.

### ISSUE THREAD

**Issue by Ericmututu** [dongzelian/SSF#9]
Hi,

Thank you for sharing this wonderful work! 

I have a question about Fig. 4 and corresponding presentation in main text, where the distributions of weights and biases seem unchanged after full fine-tuning, while SSF modulates the distributions. However, in most cases, full fine-tuning should be an upper bound compared to other parameter-efficient fine-tuning methods (except for some few-shot settings), which may indicates that we don't need to change the distribution of parameters. It seems to challenge the rationality of "the scale and shift parameters adjust the original weights and biases".

In other words, I am confused that whether we really require modulating the distribution of parameters to adapt the pre-trained model to downstream tasks for better performance? (Also, I note that SSF achieves better performance over full fine-tuning (93.99% vs. 93.82%), but this gap is slight.)

Looking forward to your reply, thanks in advance!

**Comment by Ericmututu** [comment]
I see, thank you for quick reply.
